### DISCUSSION TITLE
Poll on GUI vs Terminal

### DISCUSSION THREAD

**Discussion by Official-Husko** [Official-Husko/mul-tor#33] [Polls]
I'm currently thinking if a rewrite of this project would make sense. This also brings the question if i should now make it a GUI application or keep it a Terminal focused application.

The terminal design would probably pretty much the same.
For the GUI i have made a sketch/design a while ago (1 year ago to be honest). Its not complete but a rough idea

<details>
<summary>Images of the Design</summary>

![File Menu Default](https://github.com/Official-Husko/mul-tor/assets/27901510/c3bf4678-8c08-42fb-96f1-406d8550cc05)
![File Menu with file V1](https://github.com/Official-Husko/mul-tor/assets/27901510/0ffdb9a7-41dd-411b-b2da-0a388a9f703a)
![File Menu with File V2](https://github.com/Official-Husko/mul-tor/assets/27901510/1ac81849-0893-4a10-9fbf-e19c36992df5)

</details>

**Comment by JeOS-GH** [comment]
My opinion is that a simple GUI is a better option, especially when configuring settings. It's not that editing a text file is complicated, but I believe it's more intuitive for most people. Honestly, as long as it doesn't affect overall performance, I would prefer a GUI.

**Reply by Official-Husko** [reply]
Thank you for the vote and the explanation on what you prefer! Hopefully i can get a few more votes and opinions on this.
